src/main/java/Duke.java

```diff
@@ -10,9 +10,9 @@ public static void main(String[] args) {
         String line;
         String[] lists = new String[100];
         String[] flags = new String[100];
-        String[] dones = new String[100];
+        String[] doneTasks = new String[100];
```

Comment:
The naming for this task is not really descriptive, You can consider changing it to "taskStatus" for better clarity.